stonecrop_data_warehouse/models/marts/business/fct_capstan__orders.sql

```diff
@@ -85,31 +88,31 @@ select
 -- shipping_locations
     shipping_locations.is_active,
     shipping_locations.name,
-    shipping_locations.id,
-    shipping_locations.address_id,
+--    shipping_locations.id,
+--    shipping_locations.address_id,
     shipping_locations.shipping_location_type,
     shipping_locations.contact,
     shipping_locations.updated_at,
 -- ctd_boms
-    ctd_boms.id,
-    ctd_boms.site_augment_instance_id,
+--    ctd_boms.id,
+--    ctd_boms.site_augment_instance_id,
     ctd_boms.status,
 -- ctd_site_designs
-    ctd_site_design_instances.id,
-    ctd_site_design_instances.site_augment_instance_id,
+--    ctd_site_design_instances.id,
+--    ctd_site_design_instances.site_augment_instance_id,
     ctd_site_design_instances.status
 FROM sources.orders
-LEFT JOIN sources.phases ON phases.id = orders.phase_id
-LEFT JOIN sources.distribution_centers ON orders.distribution_center_id = distribution_centers.id
-LEFT JOIN sources.augments ON augments.id = phases.augment_id
-LEFT JOIN sources.augments_categories ON augment_categories.id = augments.augment_category_id
-LEFT JOIN sources.work_orders ON work_orders.order_id = orders.id
-LEFT JOIN sources.ctd_site_augment_instances ON ctd_site_augment_instances.id = orders.ctd_site_augment_instance_id
-LEFT JOIN sources.issues ON issues.origin_id = orders.id
-LEFT JOIN sources.issue_codes ON issue_codes.
-LEFT JOIN sources.milestones ON milestones.event_id = orders.order_id
-LEFT JOIN sources.locations ON locations.id = orders.location_id
-LEFT JOIN sources.markets ON markets.id = locations.market_id
-LEFT JOIN sources.regions ON region.id = markets.region_id
-LEFT JOIN sources.ctd_boms ON ctd_boms.id = orders.site_augment_instance_id
-LEFT JOIN sources.ctd_site_designs ON ctd_site_designs.id = orders.site_augment_instance_id
\ No newline at end of file
+LEFT JOIN {{ ref('stg_capstan__phases') }} phases ON phases.id = orders.phase_id
```

Comment:
a good by-product of explicitly naming the primary / foreign keys is that you can then join models like
```suggestion
LEFT JOIN {{ ref('stg_capstan__phases') }} phases using (phase_id)
```

which, I'm assured, is more performant.

Reply:
Okay, I updated the JOINS which I were immediately applicable. Some don't join on columns of the same name (milestones uses event_id as order ID, etc)